model/src/main/java/com/basistech/rosette/dm/RelationshipMention.java

```diff
@@ -192,8 +194,8 @@ public Double getSalience() {
                 .add("temporals", temporals)
                 .add("source", source)
                 .add("confidence", confidence)
-                .add("salience", salience);
-
+                .add("salience", salience)
+                .add("modality", modality == null ? "{}" : Iterables.toString(modality));
```

Comment:
Well, other methods that are list-valued don't bother to be as pretty as I was trying to be?